evennia/accounts/manager.py

```diff
@@ -164,9 +164,9 @@ def search_account(self, ostring, exact=True, typeclass=None):
         if typeclass:
             # we accept both strings and actual typeclasses
             if callable(typeclass):
-                typeclass = "%s.%s" % (typeclass.__module__, typeclass.__name__)
+                typeclass = f"{typeclass.__module__}.{typeclass.__name__}"
             else:
-                typeclass = "%s" % typeclass
+                typeclass = f"{str(typeclass)}"
```

Comment:
This should just be `str(typeclass`. The f-string serves no purpose here.

Reply:
Resolved.